azure_functions_devops_build/yaml/yaml_manager.py

```diff
@@ -39,14 +41,8 @@ def create_yaml(self):
             language_str = 'dotnet'
             package_route = '$(System.DefaultWorkingDirectory)/publish_output/s'
             dependencies = self._dotnet_dependencies()
-        elif self._language == JAVA:
```

Comment:
Can we move the removing Java support into a separate PR? It will keep this PR focused on removing the dependencies path. We can also easily revert that change once we do end up supporting Java. Right now it would be difficult to easily undo the "disabling Java" part of this PR since you would have to go in and manually add back the removed code (which is prone to error).

Reply:
@maiqbal11 Good point! Reverted the Java support code. Will disable Java in a separate PR.